### PR TITLE
Update plugin for round 3

### PR DESCRIPTION
**Pull request by phated**
This seems to be all that is needed to work with round 3.

I've tested most/all of the interactions in the plugin and it seems to work well. Took awhile (2-ish hours) to load all data from my 8-hour late start, but our archive node eventually spit it all out.

Ideally the contract calls would be plumbed through the new ContractsAPI stuff, but it still isn't streamlined and needs actionIDs and stuff.

### CODE REVIEW

**Review by jacobrosenthal — COMMENTED** [review]
Tests pass now

File: plugins/generated/contract.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const BROADCAST_MARKET_ADDRESS = "0xa85233C63b9Ee964Add6F2cffe00Fd84eb32338f";
\ No newline at end of file
+export const BROADCAST_MARKET_ADDRESS = "0xE6E340D132b5f46d1e472DebcD681B2aBc16e57E";
```
[the remark this answers is not in the document]
Comment (phated) [reply plugins/generated/contract.ts:1]: Gets a new dev address when we fast forward the block number. I forgot to commit mine local block number.

**Comment by phated** [comment]
Let's go! @jacobrosenthal  Want to deploy it to xdai?

**Comment by jacobrosenthal** [comment]
DF shows 188:50:38 hours until round end, Ill just use 189. Any other requests for configuration

**Comment by phated** [comment]
Sounds good. Don't think I need any other config changes.

**Comment by jacobrosenthal** [comment]
```
deploying on xdai this could take a bit
Successfully deployed to 0x05590cF3e2691538A975cbb41A6f1a814cF7FE87
Market Closes at Sun Aug 22 2021 09:02:30 GMT-0700 (Pacific Daylight Time)
FEE_PERCENT 20
CANCELLED_COUNTDOWN_BLOCKS 512
REQUEST_MAXIMUM 1000000.0
REQUEST_MINIMUM 1.2
Done in 16.61s.
```
Deployed. I think netlify deploy is still manual from you?

https://github.com/projectsophon/df-play-to-earn/tree/round-3-production

**Pull request merged**